en/docs/try-out/account-and-transaction-flow.md

```diff
@@ -172,30 +172,6 @@ user that has a `subscriber` role.
     ```
 
    The authorization code from the below URL is in the code parameter (code=`5591c5a0-14d0-3ca9-bec2-c1efe86e32ce`).
-
```

Comment:
Why we are removing the consent re-authorization section? IMO it should be there with more details on request body and steps. wDYT?

Reply:
This is directly taken from the UK documentation. We should have a separate section for consent-amendment. I'll add it in a separate PR. Improvements are tracked in https://github.com/wso2/docs-open-banking/issues/871